bedrock/stories/templates/stories/articles/art-of-engagement.html

```diff
@@ -56,16 +56,28 @@
    {
     "title": "Licensed to Build Together",
     "desc": "Who really owns the Firefox browser? Its copyleft license means we all do.",
-    "image_url": "img/stories/build-together.jpg",
-    "link_url": url('stories.articles.build-together'),
-    "image_alt": ""
+    "image": resp_img(
+      'img/stories/build-together.jpg',
+      srcset={ 'img/stories/build-together-high-res.jpg': '2x' },
+      optional_attributes={
+        'width': '349',
```

Comment:
Issue (blocking): It looks like these are missing the `card-image` class, which is used for the hover transition.

Reply:
Suggestion (non-blocking): If you want to avoid repeating the `optional_attributes`, you could also store them as a common variable:

```
{% set card_image_attr = {
    'class': 'card-image',
    'width': '349',
    'height': '239'
} %}
```

Reply:
Good catch and suggestion - thanks!